### PR TITLE
Add Phare and refactor code according to our guidelines

### PR DESCRIPTION
**Pull request by garno**
This pull request add `Phare` like in almost every gems we maintain.

### CODE REVIEW

File: emotions.gemspec [hunk 1]
```diff
@@ -24,4 +24,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'bundler', '~> 1.3'
   spec.add_development_dependency 'rspec', '~> 3.1'
   spec.add_development_dependency 'sqlite3', '~> 1.3'
+  spec.add_development_dependency 'phare', '~> 0.7'
+  spec.add_development_dependency 'rubocop', '0.27'
```
Comment (remiprev) [inline emotions.gemspec]: Is there a reason we’re using this frozen version instead of the latest `0.31` version?

File: lib/emotions.rb [hunk 2]
```diff
@@ -8,35 +8,35 @@
 require 'emotions/emotional'
 
 module Emotions
-  def self.configure
+  def configure
```
Comment (remiprev) [inline lib/emotions.rb]: I don’t think we can do that. `def self.method` in a module defines a singleton method.

File: emotions.gemspec (hunk 1 above)
Comment (garno) [inline emotions.gemspec]: No. I'll update it.

File: lib/emotions/emotional.rb [hunk 3]
```diff
@@ -136,5 +137,6 @@ def #{emotion}_about(emotive)
         RUBY
       end
     end
+    # rubocop:enable MethodLength
```
Comment (remiprev) [inline lib/emotions/emotional.rb]: I think this line is supposed to go between lines `138` and `139` (right after the `define_emotion_methods`’s `end` keyword).

**Comment by remiprev** [comment]
Looks like I *really* liked to use the explicit `self` :stuck_out_tongue_winking_eye: 

:+1:

File: lib/emotions.rb (hunk 2 above)
Comment (garno) [inline lib/emotions.rb]: Indeed, too much `self.` to remove :) It's fixed, Travis should go :green_heart:.

**Pull request merged**